### PR TITLE
drivers: dac: ltc2688: Minor fix

### PR DESCRIPTION
**Pull request by pmallick-adi**
Added a missing return statement before the error code segment.

### CODE REVIEW

File: drivers/dac/ltc2688/ltc2688.c [hunk 1]
```diff
@@ -469,6 +469,8 @@ int32_t ltc2688_init(struct ltc2688_dev **device,
 		printf("LTC2688 successfully initialized\n");
 	else
 		printf("LTC2688 initialization error (%d)\n", ret);
+	
+	return ret;
```
Comment (mchindri) [inline drivers/dac/ltc2688/ltc2688.c:473]: Put this ret inside if (!ret)
On error dev must be released.

Thanks for the fix.

File: drivers/dac/ltc2688/ltc2688.c [hunk 2]
```diff
@@ -469,6 +469,8 @@ int32_t ltc2688_init(struct ltc2688_dev **device,
 		printf("LTC2688 successfully initialized\n");
 	else
 		printf("LTC2688 initialization error (%d)\n", ret);
+
```
Comment (mchindri) [inline drivers/dac/ltc2688/ltc2688.c:472]: can you do?

Comment (mchindri) [reply drivers/dac/ltc2688/ltc2688.c:472]: This way the message is printed at each error.

Thanks for the fix.

Comment (pmallick-adi) [reply drivers/dac/ltc2688/ltc2688.c:472]: Sure will do this.

**Comment by pmallick-adi** [comment]
Change Log: 

Added changes suggested in the PR.

**Review by mchindri — APPROVED** [review]

**Pull request merged**